examples/tone_analyzer_v3.py

```diff
@@ -1,45 +1,45 @@
+from __future__ import print_function
 import json
-import os
 from os.path import join, dirname
 from watson_developer_cloud import ToneAnalyzerV3
 
 tone_analyzer = ToneAnalyzerV3(
     username='YOUR SERVICE USERNAME',
     password='YOUR SERVICE PASSWORD',
-    version='2016-05-19')
+    version='2017-09-26')
 
 print("\ntone_chat() example 1:\n")
 utterances = [{'text': 'I am very happy.', 'user': 'glenn'},
               {'text': 'It is a good day.', 'user': 'glenn'}]
 print(json.dumps(tone_analyzer.tone_chat(utterances), indent=2))
 
 print("\ntone() example 1:\n")
-print(json.dumps(tone_analyzer.tone(text='I am very happy. It is a good day.'),
-                 indent=2))
+print(json.dumps(tone_analyzer.tone(tone_input='I am very happy. It is a good day.',
+                                    content_type="text/plain"), indent=2))
 
 print("\ntone() example 2:\n")
 with open(join(dirname(__file__),
                '../resources/tone-example.json')) as tone_json:
-    tone = tone_analyzer.tone(json.load(tone_json)['text'], 'emotion')
+    tone = tone_analyzer.tone(json.load(tone_json)['text'], "text/plain")
```

Comment:
Looks like the `tones` list was removed. Based on an email thread, though, I think there's a chance it will be reintroduced to maintain backwards compatibility. Not sure if y'all want to hold off on this release until there's a resolution to that discussion.

Reply:
Good eye @glennrfisher!  The `tones` parameter was reintroduced but is now `optional`, so it moved _after_ the `content_type` parameter.  That wouldn't really be backward compatible, but fortunately we don't have to worry about that for a new major release.  And happily, this example is still correct even with `tones` being reintroduced.

@ehdsouza The reordering of the parameters might be worth mentioning in the migration guide.

Reply:
Updated the migration guide.